packages/devextreme-react/src/core/__tests__/props-updating.test.tsx

```diff
@@ -71,23 +77,31 @@ class TestComponentWithExpectation<P = any> extends TestComponent<P> {
 }
 
 describe('option update', () => {
-  it('calls option method on props update', () => {
-    const component = mount(
-      <TestComponent />,
-    );
-    expect(Widget.option.mock.calls.length).toBe(0);
+  beforeEach(() => {
+    jest.useFakeTimers();
+  })
```

Comment:
Do we need call `jest.useFakeTimers()` before each test if we called it globally?
If yes - then I think that we should remove it globally. Otherwise we need remove this `beforeEach`.